presto-common/src/main/java/com/facebook/presto/common/type/TimestampWithTimeZoneType.java

```diff
@@ -29,6 +29,12 @@ private TimestampWithTimeZoneType()
         super(parseTypeSignature(StandardTypes.TIMESTAMP_WITH_TIME_ZONE));
     }
 
+    @Override
+    public boolean valuesAreCanonical()
```

Comment:
I think something like `equalValuesAreIdentical()` would be clearer. I wouldn't know what "valuesAreCanonical" is supposed to mean without looking it up.

Reply:
Sounds reasonable to me.